dwave/embedding/transforms.py

```diff
@@ -95,6 +97,10 @@ def __init__(self, target_edges, embedding):
             if len(emb_u) != disjoint_sets[u].size(0):
                 raise DisconnectedChainError(u)
 
+    @property
+    def chain_strength(self):
+        """Returns the chain strength selected for embedding."""
```

Comment:
"Function that returns..."

Reply:
Or, if it supports this as a number, should specify the allowed types.

Reply:
Actually, according to [PEP-257](https://www.python.org/dev/peps/pep-0257/):
> The docstring is a phrase ending in a period. It prescribes the function or method's effect as a command ("Do this", "Return that"), not as a description; e.g. don't write "Returns the pathname ...".

So, just like the commit message title we talked about yesterday. :smiley:

Reply:
Though this is not really a docstring, it's an attribute description, so probably a closer example would be https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

Reply:
So probably should be `callable: Return the chain strength selected for embedding`

Reply:
Oh, I missed your last comment. Best of Guido *and* Google. Mind. Blown. :smile: